spec/models/event_spec.rb

```diff
@@ -0,0 +1,199 @@
+require 'rails_helper'
+
+RSpec.describe Event, type: :model do
+  before do
+    user = User.create(
+      provider:  'github',
+      uid:       '12345',
+      name:      'tester',
+      image_url: 'https://image.com/12345',
+    )
+
+    @event = Event.new(
+      name:     'Every Rails 輪読会',
+      place:    'discord',
+      content:  'RSpecを学習します。',
+      start_at: Time.zone.now,
+      end_at:   Time.zone.now + 1.hour,
+      owner:    user,
+    )
+  end
+
+  it 'is valid with a name, place, content, start_at, end_at' do
+    expect(@event).to be_valid
+  end
+
+  it 'is invalid without a name' do
+    event = Event.new(name: nil)
+    event.valid?
```

Comment:
```suggestion
    expect(@event).to be_invalid
```

としてあげるとよりテストコードの意図が明確になりそうです

Reply:
ありがとうございます！確かにこちらの方が何を検証しているのかわかりやすいですね。該当部分は全て `be_invalid` で検証するように修正しました！

該当コミット：https://github.com/ymmtd0x0b/awesome_events/pull/3/commits/a717871dc798f88b04e0d8f8c7ac5adf932187d8